compiler/rustc_session/src/options.rs

```diff
@@ -384,7 +384,7 @@ mod desc {
     pub const parse_mir_spanview: &str = "`statement` (default), `terminator`, or `block`";
     pub const parse_dump_mono_stats: &str = "`markdown` (default) or `json`";
     pub const parse_instrument_coverage: &str =
-        "`all` (default), `except-unused-generics`, `except-unused-functions`, or `off`";
+        "`all` (default), `branch`, `except-unused-generics`, `except-unused-functions`, or `off`";
```

Comment:
We should also update this error message to include `branch`:

https://github.com/rust-lang/rust/blob/3fbcfd2b6f7030cb70328aa759107efc1516912c/compiler/rustc_session/src/config.rs#L2708